OscLib/IOscCalcSterile.h

```diff
@@ -0,0 +1,76 @@
+#ifndef OSC_IOSCCALCSTERILE_H
+#define OSC_IOSCCALCSTERILE_H
+
+//////////////////////////////////////////////////////////////////////////
+//                                                                      //
+// \file IOscCalcSterile.h                                              //
+//                                                                      //
+// Base class for sterile oscillation calculator                        //
+// <jhewes15@fnal.gov>                                                  //
+//                                                                      //
+//////////////////////////////////////////////////////////////////////////
+
+#include "OscLib/IOscCalc.h"
+
+namespace osc
+{
+  class IOscCalcSterile: public IOscCalcAdjustable
```

Comment:
Is it worth a comment somewhere here about what pdg code of zero means in the context of a sterile calculator?

Reply:
Added some extra comments here to clarify